sdk/src/utils.ts

```diff
@@ -271,3 +271,20 @@ export const getDeploymentData = async (
     throw new Error(`${contractName}.json or ${contractName}.wasm not found`)
   }
 }
+
+export const DENOMINATOR = BigInt(Math.pow(10, 12))
+export const PRICE_DENOMINATOR = BigInt(Math.pow(10, 24))
```

Comment:
Best solution will be to extend scale macro: https://github.com/invariant-labs/protocol-a0/blob/master/sdk/math/wasm_helpers.rs#L56
Add denominator similarity as was added `get${decimal}Scale`.